### PR TITLE
docs(ui-toolkit): update bundles

### PR DESCRIPTION
**Pull request by bwendlandt-intel**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

- Update styling of KVM/SOL [edit:ALL] bundle docs to match docs
- Update code snippets to match 1.1 or newer changes i.e. server -> mpsServer
- Initial formatting fix for localization.md

## Anything the reviewer should know when reviewing this PR?
Bundles outside of KVM and SOL are being deprecated, correct? [Edit: Starting in v1.3.0, others will be deprecated]

### CODE REVIEW

File: docs/UIToolkit/Bundles/CIRAConfigs.md [hunk 1]
```diff
@@ -1,91 +1,100 @@
...
+	cd ui_toolkit
+	```
+
+4. Install the dependencies.
+	```
+	npm install
+	```
 
-## Customize the control
+## Run in Development Enviroment
```
Comment (kimberlyxdavis) [inline docs/UIToolkit/Bundles/CIRAConfigs.md:39]: Environment, with an "n"

File: docs/UIToolkit/Bundles/CIRAConfigs.md [hunk 2]
```diff
@@ -1,91 +1,100 @@
...
 
-To add new changes and test the changes  instantly before bundling the control, webpack dev server can be used
+To add and test new changes before bundling the control, a webpack dev server can be used.
```
Comment (kimberlyxdavis) [inline docs/UIToolkit/Bundles/CIRAConfigs.md:41]: To add and test new changes before bundling the control, use a webpack dev server.

File: docs/UIToolkit/Bundles/DeviceGrid.md [hunk 3]
```diff
@@ -1,87 +1,106 @@
...
-
-## Add a new Language for Internationalization
-
-Please refer to [Localization](../localization.md) docs
+
+- [Git](https://git-scm.com/)
+- [Visual Studio Code](https://code.visualstudio.com/) or any other IDE 
+- [Node.js](https://nodejs.org/)
+- [Chrome* Browser](https://www.google.com/chrome)
+- [A MPS Server with an AMT Device Connected](../../../Docker/dockerLocal/)
```
Comment (kimberlyxdavis) [inline docs/UIToolkit/Bundles/DeviceGrid.md:17]: "An MPS Server" or just "MPS Server"

File: docs/UIToolkit/Bundles/DeviceGrid.md [hunk 4]
```diff
@@ -1,87 +1,106 @@
...
+	```
+	cd ui_toolkit
+	```
+
+4. Install the dependencies.
+	```
+	npm install
+	```
+
+## Run in Development Enviroment
```
Comment (kimberlyxdavis) [inline docs/UIToolkit/Bundles/DeviceGrid.md:39]: "Enviroment" should be "Environment"

File: docs/UIToolkit/Bundles/CIRAConfigs.md [hunk 5]
```diff
@@ -1,91 +1,100 @@
...
+2. In a Terminal (Linux) or Command Prompt (Windows), navigate to the `ui-toolkit` directory.
 
-## Add a new Language for Internationalization
+3. Serve the HTML page.
+	```
+	npx serve
+	```
 
- Please refer to [Localization](../localization.md) docs
+4. Open a new Chrome* browser and navigate to the following url:
```
Comment (kimberlyxdavis) [inline docs/UIToolkit/Bundles/CIRAConfigs.md:96]: URL

File: docs/UIToolkit/Bundles/CIRAConfigs.md (hunk 2 above)
Comment (kimberlyxdavis) [reply docs/UIToolkit/Bundles/CIRAConfigs.md:41]: I know it sounds trivial, but the way it's written it sounds as if the web dev server is driving the action when it's the user.

File: docs/UIToolkit/Bundles/DeviceGrid.md [hunk 6]
```diff
@@ -1,87 +1,106 @@
...
+	```
+
+4. Open a new Chrome* browser and navigate to the following url:
+	```
+	http://localhost:5000/src/sample/sampleDG.htm?deviceId=[AMT-Device-GUID]&mpsServer=https://[MPS-Server-IP-Address]:3000
+	```
+
+You will see the errors in the following scenarios: 
+
+- Compilation errors if the ui-toolkit was not downloaded and installed into your react app.
```
Comment (kimberlyxdavis) [inline docs/UIToolkit/Bundles/DeviceGrid.md:103]: I would make all these complete sentences or fragments.

File: docs/UIToolkit/localization.md [hunk 7]
```diff
@@ -1,31 +1,46 @@
...
+
+	Rebuild and generate a new bundle before testing the changes.
 
 Language can be changed in the browser under langauage section of the browser settings. English is the default if no customized translation file provided for the langauage.
 
-## Get the localised strings for the web consoles with localization enabled.
+## Get Localized Strings for Web Consoles with Localization Enabled
 
-If users web console is already enabled with localization please make sure to add the [translations](https://github.com/open-amt-cloud-toolkit/ui-toolkit/tree/master/public/locales) of UI-controls into your web consoles translations file. 
+If a user's web console is already enabled with localization, please make sure to add the [translations](https://github.com/open-amt-cloud-toolkit/ui-toolkit/tree/master/public/locales) of UI-controls into your web console's translations file.
```
Comment (kimberlyxdavis) [inline docs/UIToolkit/localization.md:46]: If your web console is already enabled with localization, please make sure to add the [translations](https://github.com/open-amt-cloud-toolkit/ui-toolkit/tree/master/public/locales) of UI-controls into your web console's translations file.

**Review by kimberlyxdavis — CHANGES_REQUESTED** [review]
Many of these smallish errors got propagated throughout the multiple files. I didn't call them out each time. You'll see.

**Review by kimberlyxdavis — APPROVED** [review]

**Review by madhavilosetty-intel — APPROVED** [review]

**Pull request merged**